src/core/CMakeLists.txt

```diff
@@ -66,6 +66,7 @@ target_link_libraries(core PRIVATE warnings)
 
 set_property(TARGET core PROPERTY FOLDER "libraries")
 
-if (MSVC)
+# NOTE: /analyze throws spurious warnings when building with the Ninja generator in VS2019
```

Comment:
What type of warnings are getting thrown? Can we just selectively ignore them in this configuration instead?

Reply:
These trigger c28251 in a number of declarations in intrin0.h. I can refactor the change to surgically ignore this warning for that file after work today. Thanks for the suggestion

Reply:
Thanks! I'd much rather turn off just 1 warning than them all.

Reply:
Makes complete sense. I probably should have taken the time to do it honestly. I tend to get impatient when working through build related issues ':)